wolfssl/test.h

```diff
@@ -1681,6 +1687,21 @@ static WC_INLINE int myVerify(int preverify, WOLFSSL_X509_STORE_CTX* store)
                                                                   subject);
         XFREE(subject, 0, DYNAMIC_TYPE_OPENSSL);
         XFREE(issuer,  0, DYNAMIC_TYPE_OPENSSL);
+#if defined(SHOW_CERTS) && !defined(NO_FILESYSTEM)
```

Comment:
This just ends up printing out multiple copies of the certificate. Mainly because the `myVerify` callback is issued for every certificate in the chain (due to `--enable-opensslextra` setting `WOLFSSL_ALWAYS_VERIFY_CB` and `WOLFSSL_VERIFY_CB_ALL_CERTS`. I think it would make more sense to call this code in the example client when `-j` is not set and `OPENSSL_EXTRA` and `SHOW_CERTS` are defined.